common/rdm/PidStoreHelper.cpp

```diff
@@ -286,6 +288,15 @@ void PidStoreHelper::SupportedPids(
   if (store) {
     store->AllPids(descriptors);
   }
+
+  // Sort PIDs by name
+  sort(descriptors->begin(), descriptors->end(), &CompPidsForSort);
```

Comment:
Given something else may want the PIDs sorted by ID for example, or might not care, I think it would be better to do the sort in the consuming function.

Reply:
I think I misinterpreted your previous comment on this one.  Will move sorting to the consumer.